sources/packages/forms/src/form-definitions/studentaccountactivity.json

```diff
@@ -0,0 +1,10081 @@
+                  "input": false,
+                  "tableView": false,
+                  "placeholder": "",
+                  "prefix": "",
+                  "customClass": "",
+                  "suffix": "",
+                  "multiple": false,
+                  "defaultValue": null,
+                  "protected": false,
+                  "unique": false,
+                  "persistent": false,
+                  "hidden": false,
+                  "clearOnHide": true,
+                  "refreshOn": "",
+                  "redrawOn": "",
+                  "modalEdit": false,
+                  "dataGridLabel": false,
+                  "labelPosition": "top",
+                  "description": "",
+                  "errorLabel": "",
+                  "tooltip": "",
+                  "hideLabel": false,
+                  "tabindex": "",
+                  "disabled": false,
+                  "autofocus": false,
+                  "dbIndex": false,
+                  "customDefaultValue": "",
+                  "calculateValue": "",
+                  "calculateServer": false,
+                  "widget": null,
+                  "attributes": {},
+                  "validateOn": "change",
+                  "validate": {
+                    "required": false,
+                    "custom": "",
+                    "customPrivate": false,
+                    "strictDateValidation": false,
+                    "multiple": false,
+                    "unique": false
+                  },
+                  "conditional": {
+                    "show": null,
+                    "when": null,
+                    "eq": ""
+                  },
+                  "overlay": {
+                    "style": "",
+                    "left": "",
+                    "top": "",
+                    "width": "",
+                    "height": ""
+                  },
+                  "allowCalculateOverride": false,
+                  "encrypted": false,
+                  "showCharCount": false,
+                  "showWordCount": false,
+                  "properties": {},
+                  "allowMultipleMasks": false,
+                  "addons": [],
+                  "tag": "p",
+                  "id": "exx8mcu"
+                },
+                {
+                  "label": "Active restriction codes",
+                  "labelPosition": "top",
+                  "labelWidth": "",
+                  "labelMargin": "",
+                  "description": "",
+                  "tooltip": "",
+                  "disableAddingRemovingRows": true,
+                  "conditionalAddButton": "",
+                  "reorder": false,
+                  "addAnother": "",
+                  "addAnotherPosition": "bottom",
+                  "layoutFixed": false,
+                  "enableRowGroups": false,
+                  "initEmpty": false,
+                  "customClass": "formio-datagrid-unset student-activity-grid mt-n2",
```

Comment:
Were you able to reduce the spacing between the list items? I noticed that the print on the PR description has only one item. How does it look when there are 2 items.

Reply:
I did not try further to reduce the spacing between the items. Used the existing class which was present. Added and additional style for bullet left padding. 
![image](https://user-images.githubusercontent.com/54600590/171232645-f36c2c58-7449-4e74-91c9-be37b3d3e727.png)